scripts/aspenmesh-collector-setup.sh

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env bash
+
+set -Eeuo pipefail
+
+if ! [ -x "$(command -v helm)" ]; then
+  echo "helm is not installed. Please visit https://helm.sh/docs/intro/install/" >&2
+  exit 1
+fi
+
+usage() {
+  echo "
+usage: ${0} [flags] [-h|--help]
+Flags               Purpose
+--apiKey            Required; The FusionAuth API key for the user's group
+--endpoint          Optional; The Promscale remote write endpoint. Default value: https://metrics.cloud.aspenmesh.io/write
+" >&2
+
+    exit 1
+}
+
+apiKey=""
+endpoint="https://metrics.cloud.aspenmesh.io/write"
```

Comment:
same here - remove

Reply:
We need this for testing

Reply:
Suggestion to make this less obvious:

```suggestion
endpoint="${AM_METRICS_ENDPOINT:-https://metrics.cloud.aspenmesh.io/write}"
```